src/ImageRunModal.jsx

```diff
@@ -635,6 +590,61 @@ export class ImageRunModal extends React.Component {
         return owner === systemOwner;
     };
 
+    isFormValid = (validationFailed) => {
+        return Object.keys(validationFailed).length == 0;
+    };
+
+    async validateContainerName(containerName) {
+        try {
+            await client.containerExists(this.isSystem(), containerName);
+        } catch (error) {
+            return;
+        }
+        return _("Name already in use");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230725-110243-108e1b8a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)